packages/core/admin/admin/src/pages/HomePage/ContentBlocks.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { Stack } from '@strapi/parts/Stack';
+import ReadDoc from '@strapi/icons/ReadDoc';
+import CodeExample from '@strapi/icons/CodeExample';
+import Tutorial from '@strapi/icons/Tutorial';
+import Blog from '@strapi/icons/Blog';
+import { ContentBox } from '@strapi/helper-plugin';
+
+const BlockLink = styled.a`
+  text-decoration: none;
+`;
+
+const ContentBlocks = () => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Stack size={5}>
+      <BlockLink href="https://strapi.io/resource-center" target="_blank">
```

Comment:
Can you add this attribute ` rel="noopener noreferrer nofollow" ` ?  [Here](https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/)  is some info about this attribute.